README.md

```diff
@@ -46,6 +47,81 @@ npx truffle compile
 npm run networks-inject 
 ```
 
+# Deploying a simple market maker scenario to Rinkeby:
+
+The following script does deploy a simple order market maker order and a relevant
+```
+// Get token id of DAI
+npx truffle exec scripts/stablex/invokeViewFunction.js 'tokenAddressToIdMap' '0x5592EC0cfb4dbc12D3aB100b257153436a1f0FEa' --network rinkeby
+
+//export the result
+export TOKEN_ID_DAI=[Result from last call]
+
+// Get token id of TrueUSD
+npx truffle exec scripts/stablex/invokeViewFunction.js 'tokenAddressToIdMap' '0x0000000000085d4780B73119b644AE5ecd22b376' --network rinkeby
+
+//export the result
+export TOKEN_ID_TUSD=[Result from last call]
+
+// Make sure there is enough funds in the exchange for the intended trades
+npx truffle exec scripts/stablex/deposit.js --accountId=0 --tokenId=0 --amount=30 --network rinkeby&& \
+npx truffle exec scripts/stablex/deposit.js --accountId=0 --tokenId=$TOKEN_ID_TUSD --amount=100 --network rinkeby
+
+// Place market maker order in current auction
+npx truffle exec scripts/stablex/place_order.js --accountId=0 --buyToken=$TOKEN_ID_DAI --sellToken=$TOKEN_ID_TUSD --minBuy=1000 --maxSell=998 --validFor=20 --network rinkeby
```

Comment:
Maybe a noob question since I have not worked directly with these scripts, but does specifying `validFor` place and order which is valid for the next 20 batches? In other words, is the validity interval = `[currentBatchId, currentBatchId + 20]`?

Reply:
yep